docs/public/SUMMARY.md

```diff
@@ -2,6 +2,7 @@
  https://rust-lang.github.io/mdBook/format/summary.html -->
 # Documents in public document tree
 
+- [Zonemaster documentation](README.md)
```

Comment:
Maybe rename this to Introduction? That's more precise.

```suggestion
- [Introduction](README.md)
```

Reply:
The README.md does not contain any introduction as far as I can see.

Reply:
How about "Overview" then? I'm just trying to get away from the situation where we have a "documentation" section right next to other sections that also contain documentation.

I had a look around at what other mdbooks name their first pages and "Introduction" seemed to be the most popular, and that's why I suggested it. But I do agree our page doesn't contain much introductory material. Maybe we should have an introduction page?